engine/Container.js

```diff
@@ -205,7 +221,7 @@ export class Container extends InkObject{//also implements INamedContent. Not su
 				return foundContent;
 			}
 			else {
-				throw new StoryException("Content '"+component.name+"' not found at path: '"+this.path+"'");
+				throw null;
```

Comment:
[I think we just want to return `null` here](https://github.com/inkle/ink/compare/0.7.5...0.8.1#diff-ebd46ea8e5c494f6b2a75971f9ad2badR224), not throw it :)

Reply:
Totally, that's a typo! :)